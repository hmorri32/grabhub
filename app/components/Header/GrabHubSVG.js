import React from 'react';

export const GrabHubSVG = () => (
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
    <path d="M26.2 35.4l-20 12.5v4.2l20 12.5 2.6-4.2L12.2 50l16.6-10.4zM73.8 35.4l-2.6 4.2L87.8 50 71.2 60.4l2.6 4.2 20-12.5v-4.2z" />
    <circle cx="50" cy="50" r="5" />
    <circle cx="65" cy="50" r="5" />
    <circle cx="35" cy="50" r="5" />
  </svg>
);
