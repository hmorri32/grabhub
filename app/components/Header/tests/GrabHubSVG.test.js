import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { GrabHubSVG } from '../GrabHubSVG';

describe('<GrabHubSVG />', () => {
  it('should match the snapshot', () => {
    const renderedComponent = renderer.create(<GrabHubSVG />).toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
});
