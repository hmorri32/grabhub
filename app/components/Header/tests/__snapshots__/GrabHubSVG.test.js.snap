// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GrabHubSVG /> should match the snapshot 1`] = `
<svg
  viewBox="0 0 100 100"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M26.2 35.4l-20 12.5v4.2l20 12.5 2.6-4.2L12.2 50l16.6-10.4zM73.8 35.4l-2.6 4.2L87.8 50 71.2 60.4l2.6 4.2 20-12.5v-4.2z"
  />
  <circle
    cx="50"
    cy="50"
    r="5"
  />
  <circle
    cx="65"
    cy="50"
    r="5"
  />
  <circle
    cx="35"
    cy="50"
    r="5"
  />
</svg>
`;
